mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -236,19 +268,19 @@
         <Flags index="IgnoreUnitCost" value="1"/>
     </CAbilMorph>
     <CAbilMorph id="BurrowZerglingUp">
+        <CmdButtonArray index="Execute">
+            <Flags index="ShowInGlossary" value="0"/>
```

Comment:
**undocumented**: 

Pointless change. How did this make it into the xml? The value before was 0 already.
ShowInGlossary 0 means that the ability should not be listed in the unit's abilities in the game's help menu (F12).
The game did not list it before.

=> should be removed because nothing is changed